### PR TITLE
DisplayProperties: Added a menubar

### PR DESCRIPTION
**Pull request by Quaker762**
Added a menubar to the `DisplayProperties` application to make it
more consistent with the other programs in the system.

### CODE REVIEW

File: Applications/DisplayProperties/main.cpp [hunk 1]
```diff
@@ -12,12 +16,28 @@ int main(int argc, char** argv)
...
+
+    auto app_menu = GMenu::construct("Display Properties");
+    app_menu->add_action(GAction::create("Exit", { Mod_Alt, Key_F4 }, [&](const GAction&) {
+        app.quit();
+    }));
+    menubar->add_menu(move(app_menu));
+
+    auto help_menu = GMenu::construct("Help");
+    help_menu->add_action(GAction::create("About", [&](const GAction&) {
+        GAboutDialog::show("Terminal", load_png("/res/icons/32x32/app-display-properties.png"), window);
```
Comment (awesomekling) [inline Applications/DisplayProperties/main.cpp]: "Terminal"? ;)

Comment (Quaker762) [reply Applications/DisplayProperties/main.cpp]: Whoops, I was using `Terminal`'s `main.cpp` for inspiration. Looks like I verbatim copied it.

**Comment by bugaevc** [comment]
[whine mode on]
Please, commit messages should be imperative, "Add a menubar", not "Added" :(

**Comment by Quaker762** [comment]
> [whine mode on]
> Please, commit messages should be imperative, "Add a menubar", not "Added" :(

Any particular reason?

**Comment by bugaevc** [comment]
Consistency with other commit messages, mostly (including ones generated by Git itself, such as for merges). Just as we follow @awesomekling's preferred C++ style, we should be following his preferred Git commit message style 😀

Refer to this guide for more guidelines: https://chris.beams.io/posts/git-commit/

File: Applications/DisplayProperties/main.cpp [hunk 2]
```diff
@@ -12,12 +16,28 @@ int main(int argc, char** argv)
...
     window->resize(400, 448);
     window->set_resizable(false);
     window->set_main_widget(instance.root_widget());
     window->set_icon(load_png("/res/icons/16x16/app-display-properties.png"));
 
+    // Let's create the menubar first
+    auto menubar = make<GMenuBar>();
+
+    auto app_menu = GMenu::construct("Display Properties");
+    app_menu->add_action(GAction::create("Exit", { Mod_Alt, Key_F4 }, [&](const GAction&) {
```
Comment (awesomekling) [inline Applications/DisplayProperties/main.cpp]: Please use `GCommonActions` for this:
```suggestion
    app_menu->add_action(GCommonActions::make_quit_action([&](const GAction&) {
```

Comment (Quaker762) [reply Applications/DisplayProperties/main.cpp]: Oh that's super cool. I'll get on updating all of the files that use the original version (I think `Terminal` still does)

Comment (awesomekling) [reply Applications/DisplayProperties/main.cpp]: I don't think there are any :)

Comment (Quaker762) [reply Applications/DisplayProperties/main.cpp]: > I don't think there are any :)

Must be an issue with my fork, you know that me and git don't mix that well ;)

**Comment by awesomekling** [comment]
Okay! :)

**Pull request merged**